reconcile/quay_permissions.py

```diff
@@ -66,17 +84,28 @@ def run(dry_run):
                             logging.warning('wrong service kind, ' +
                                             'should be quay-membership')
                             continue
-                        if permission['org'] != org_name:
+
+                        perm_org_key = (
+                          permission['quayOrg']['instance']['name'],
+                          permission['quayOrg']['name']
+                        )
+
+                        if perm_org_key != org_key:
                             logging.warning('wrong org, ' +
-                                            f'should be {org_name}')
+                                            f'should be {org_key}')
```

Comment:
You should use the `%s` plus args for logging messages, so the string only gets formatted when the message is actually generated.

Reply:
fixed